pkg/controller/master/setting/ntp_server.go

```diff
@@ -0,0 +1,91 @@
+package setting
+
+import (
+	"reflect"
+	"strings"
+
+	nodev1 "github.com/harvester/node-manager/pkg/apis/node.harvesterhci.io/v1beta1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/util"
+)
+
+const (
+	nodeConfigNS = "harvester-system"
+)
+
+func (h *Handler) syncNTPServer(setting *harvesterv1.Setting) error {
+	logrus.Infof("Processing setting %s, value: %s", setting.Name, setting.Value)
+
+	ntpServers := strings.Replace(setting.Value, ",", " ", -1)
+	if ntpServers == "" {
```

Comment:
In which scenario, the ntpServers will be empty?

Reply:
I do not see any chance that the `ntpServers` will be empty.
But I am afraid something I do not consider more, so I am adding this check here.